spec/controllers/follows_controller_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe FollowsController, type: :controller do
+  sign_in_user
+
+  let(:other_user) { create(:user) }
+  let(:question) { create(:question, user: @user) }
+  let(:follow) { create(:follow, user: @user, followable_type: "Question", followable_id: question.id) }
+  let(:other_follow) { create(:follow, user: other_user, followable_type: "Question", followable_id: question.id) }
+
+  describe 'POST #create' do
+    context 'correct work' do
+      it 'create object' do
+        expect { post :create, params: { user_id: @user, question_id: question, format: :js } }.to change(question.follows, :count).by(1)
+        expect { post :create, params: { user_id: @user, question_id: question, format: :js } }.to change(@user.follows, :count).by(1)
```

Comment:
You're creating two subscriptions for the same user-question, aren't you? That should not work.

And it's better to use compound matcher:

```ruby
expect { ... }.to change(...).by(...).and change(...).by(...)
```